2019-12-04 05:12:11 INFO zookeeper_session.py:143 create_zeus_entity.py is attempting to connect to Zookeeper
2019-12-04 05:43:02 INFO zookeeper_session.py:143 create_zeus_entity.py is attempting to connect to Zookeeper
2019-12-04 21:03:48 INFO zookeeper_session.py:143 create_zeus_entity.py is attempting to connect to Zookeeper
2019-12-04 22:37:38 INFO zookeeper_session.py:143 create_zeus_entity.py is attempting to connect to Zookeeper
2019-12-05 00:40:18 INFO zookeeper_session.py:143 create_zeus_entity.py is attempting to connect to Zookeeper
2019-12-05 00:41:55 INFO zookeeper_session.py:143 create_zeus_entity.py is attempting to connect to Zookeeper
2019-12-05 19:09:23 INFO zookeeper_session.py:143 create_zeus_entity.py is attempting to connect to Zookeeper
2019-12-05 19:16:01 INFO zookeeper_session.py:143 create_zeus_entity.py is attempting to connect to Zookeeper
2019-12-05 19:16:33 INFO zookeeper_session.py:143 create_zeus_entity.py is attempting to connect to Zookeeper
